autoscheduler/frontend/src/components/SchedulingPage/Schedule/MeetingCard/MeetingCard.tsx

```diff
@@ -23,41 +28,75 @@ const MeetingCard: React.FC<MeetingCardProps> = ({
     startTimeHours, startTimeMinutes, endTimeHours, endTimeMinutes, section, meetingType,
   } = meeting;
   const [isBig, setIsBig] = React.useState(true);
+  const [snackbarMessage, setSnackbarMessage] = React.useState('');
+  const dispatch = useDispatch();
 
   // hide meeting type if the card is small
   const handleResize = React.useCallback((newVal: boolean): void => {
     setIsBig(newVal);
   }, []);
 
+  const onClick = (): void => {
```

Comment:
The proper name for react event handler functions is `handleX` rather than `onX` (you can see in the react documentation and our other event handlers): you can leave the prop as `onClick`, but when setting its value here it should be `handleClick`.
```suggestion
  const handleClick = (): void => {
```